config/samples/mcpserver-test-servers.yaml

```diff
@@ -115,6 +115,21 @@ spec:
 ---
 apiVersion: mcp.kagenti.com/v1alpha1
 kind: MCPServer
+metadata:
+  name: conformance-server
+  namespace: mcp-test
+  labels:
+    'kagenti/mcp': 'true'
+spec:
+  # toolPrefix: conformance_ # conformance test suite expects no prefix
```

Comment:
no prefix makes sense.
Noting I had issues with any tools that started with `test_` from the conformance server as they conflict with the test1 server which has a prefix of `test_`. 
Let's change that server to use test1_ prefix